setup.py

```diff
@@ -91,7 +91,7 @@
 
     # You can just specify the packages manually here if your project is
     # simple. Or you can use find_packages().
-    packages=find_packages(exclude=['contrib', 'docs', '*tests*']),
+    packages=find_packages(exclude=['contrib', 'docs']),
```

Comment:
Unfortunately this cannot be done: it would insert the "test" folder in the wheel. I'll check if I can remove this and still have your contribution work.

Reply:
Is that a problem? Many packages I work with (NumPy, SciPy, etc.) ship their `tests` folder.